compliance_checker/_version.py

```diff
@@ -41,7 +41,7 @@ def get_config():
     cfg = VersioneerConfig()
     cfg.VCS = "git"
     cfg.style = "pep440"
-    cfg.tag_prefix = "v"
+    cfg.tag_prefix = ""
```

Comment:
My guess is that this is OK but you should make that change in the `setup.cfg` and re-generate this file. Last time I did this I just ran `versioneer install` again, maybe they added an `update` option now.

Reply:
setup.cfg was updated in the last PR but I forgot to commit this file after running ```python versioneer.py install```

Reply:
This should be OK then.